### PR TITLE
SafeNet Event Collector

### PR DESCRIPTION
**Pull request by daryakoval**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-2833

## Description
Safe Net Event Collector

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.8.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@dantavori Doc review completed.

**Review by dantavori — COMMENTED** [review]
looks good, let's schedule a demo

File: Packs/SafeNet_Trusted_Access/ModelingRules/SafeNetTrustedAccessModelingRules/SafeNetTrustedAccessModelingRules.xif [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    XDM.Audit.TriggeredBy.identity.name = json_extract_scalar(context, "$.principalId"),
+    XDM.Audit.original_event_type = json_extract_scalar(details, "$.type"),
+    XDM.Audit.original_event_description = json_extract_scalar(details, "$.description"),
+    XDM.Audit.operation_type = json_extract_scalar(details, "$.operationType"),
+    XDM.Audit.audited_resource.type = json_extract_scalar(details, "$.operationObjectType"),
+    XDM.Audit.audited_resource.name = json_extract_scalar(details, "$.operationObjectName");
+
+
+[MODEL: dataset="safenet_trusted_access_raw", model="Auth"]
+filter json_extract_scalar(details, "$.type") in ("AUTHENTICATION", "ACCESS_REQUEST")
```
[the remark this answers is not in the document]
Comment (daryakoval) [reply Packs/SafeNet_Trusted_Access/ModelingRules/SafeNetTrustedAccessModelingRules/SafeNetTrustedAccessModelingRules.xif:16]: There are only 3 types. @idovandijk told.

[the remark this answers is not in the document]
Comment (idovandijk) [reply Packs/SafeNet_Trusted_Access/ModelingRules/SafeNetTrustedAccessModelingRules/SafeNetTrustedAccessModelingRules.xif:16]: Design partner asked for those 3

[the remark this answers is not in the document]
Comment (idovandijk) [reply Packs/SafeNet_Trusted_Access/ModelingRules/SafeNetTrustedAccessModelingRules/SafeNetTrustedAccessModelingRules.xif:16]: https://jira-hq.paloaltonetworks.local/browse/CIAC-2833?jql=project%20%3D%20CIAC%20AND%20text%20~%20%22safenet%22

File: Packs/SafeNet_Trusted_Access/Integrations/SafeNetTrustedAccessEventCollector/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+Retrieve access, authentication, and audit logs and stores them in a Security Information and Event Management (SIEM) system, local repository, or syslog file server. You can retrieve the logs only for the tenant that is associated with the API key, or for a direct or delegated child of that tenant.
```
[the remark this answers is not in the document]
Comment (daryakoval) [reply Packs/SafeNet_Trusted_Access/Integrations/SafeNetTrustedAccessEventCollector/README.md:1]: yes

**Review by dantavori — APPROVED** [review]

**Pull request merged**